File: api/core/src/main/kotlin/data/TestData.kt
package data

import domain.Status
import domain.Test
import kotlinx.serialization.Serializable

@Serializable
data class TestData(
    val test: Test,
    val status_refs: List<Status> = listOf(),
    val naloga_refs: List<NalogaData> = listOf()
)

File: app/src/app/services/api/openapi/models/test-data.ts
/* tslint:disable */
/* eslint-disable */
import { NalogaData } from './naloga-data';
import { Status } from './status';
import { Test } from './test';
export interface TestData {
  naloga_refs?: Array<NalogaData>;
  status_refs?: Array<Status>;
  test?: Test;
}
